CompilerSource/settings-parse/parse_ide_settings.cpp

```diff
@@ -156,11 +164,16 @@ void parse_ide_settings(const char* eyaml)
 #if CURRENT_PLATFORM_ID == OS_WINDOWS
   eobjs_directory = myReplace(escapeEnv(eobjs_directory), "\\","/");
   codegen_directory = myReplace(escapeEnv(codegen_directory), "\\","/");
+  enigma_root = myReplace(escapeEnv(enigma_root), "\\","/");
 #else
   eobjs_directory = escapeEnv(eobjs_directory);
   codegen_directory = escapeEnv(codegen_directory);
+  enigma_root = escapeEnv(enigma_root);
 #endif
 
+  if (enigma_root.back() != '/')
```

Comment:
This stuff could probably also use to be fs::path, unless there's a C++17 chasm somewhere in the source tree.

Reply:
theres a lot to be redone here but I didnt want to do that yet